jme3-terrain/src/main/resources/Common/MatDefs/Terrain/PBRTerrain.j3md

```diff
@@ -0,0 +1,366 @@
+// NOTE: Doesn't support OpenGL1
+MaterialDef PBR Terrain {
+
+    MaterialParameters {
+
+
+        Boolean UseVertexColorsAsSunIntensity //set true to make the vertex color's R channel how exposed a vertex is to the sun
+        Float StaticSunIntensity              //used for setting the sun exposure value for a whole material
+                                             //these are usually generated at run time or setup in a level editor per-geometry, so that models indooes can have the DirectionalLight dimmed accordingly.
+
+        Boolean BrightenIndoorShadows  //set true if shadows are enabled and indoor areas withour full sun exposure are too dark compared to when shadows are turned off in settings
+
+
+        Int PlaguedMapScale : 8
+
+        Float AfflictionRoughnessValue : 1.0
+        Float AfflictionMetallicValue : 0.0
+        Float AfflictionEmissiveValue : 0.0 //note that this is simplified into one value, rather than 2 with power and intensity like the regular pbr values.
+
+        Texture2D AfflictionTexture
+        Texture2D PlaguedAlbedoMap  -LINEAR
+        Texture2D PlaguedNormalMap  -LINEAR
+        Texture2D PlaguedRoughnessMetallicMap -LINEAR
+        Texture2D PlaguedEmissiveMap -LINEAR
+
+        Color AfflictionEmissiveColor : 1.0 1.0 1.0 1.0
+
+        //used for decal mapping with afflictionMap, in order to convert world coords to tex coords so afflictionTexture accurately represents the world
+        Float TileWidth : 0
+        Vector3 TileLocation
+
+        Int AfflictionMode_0 : 1
+        Int AfflictionMode_1 : 1
+        Int AfflictionMode_2 : 1
+        Int AfflictionMode_3 : 1
+        Int AfflictionMode_4 : 1
+        Int AfflictionMode_5 : 1
+        Int AfflictionMode_6 : 1
+        Int AfflictionMode_7 : 1
+        Int AfflictionMode_8 : 1
+        Int AfflictionMode_9 : 1
+        Int AfflictionMode_10 : 1
+        Int AfflictionMode_11 : 1
+
+        Float Roughness_0 : 0.0
+        Float Roughness_1 : 0.0
+        Float Roughness_2 : 0.0
+        Float Roughness_3 : 0.0
+        Float Roughness_4 : 0.0
+        Float Roughness_5 : 0.0
+        Float Roughness_6 : 0.0
+        Float Roughness_7 : 0.0
+        Float Roughness_8 : 0.0
+        Float Roughness_9 : 0.0
+        Float Roughness_10 : 0.0
+        Float Roughness_11 : 0.0
+
+        Float Metallic_0 : 0.0
+        Float Metallic_1 : 0.0
+        Float Metallic_2 : 0.0
+        Float Metallic_3 : 0.0
+        Float Metallic_4 : 0.0
+        Float Metallic_5 : 0.0
+        Float Metallic_6 : 0.0
+        Float Metallic_7 : 0.0
+        Float Metallic_8 : 0.0
+        Float Metallic_9 : 0.0
+        Float Metallic_10 : 0.0
+        Float Metallic_11 : 0.0
+
+
+        Int DebugValuesMode

```

Comment:
Could we give this a debug value?

Reply:
I could give DebuValuesMode a default value of -1; this would have the same functionality as leaving it undefined. 

As is, DebugValuesMode is intended to be -1 or undefined until the user needs to inspect the albedo/normal/roughness/metallic/ao outputs. The albedo debug mode can also be used to render in an unshaded mode, but is best disabled by default.

Reply:
Leave it as undefined, then. But DebugValuesMode is a feature we'll need to document at some point.